deploy/.gitignore

```diff
@@ -0,0 +1 @@
+/.gotools/
```

Comment:
do we need first `/`?
```suggestion
.gotools/
```

Reply:
Leading `/` was intentional, means "just in this dir". But thinking about it, it makes sense to remove ignore it everywhere...

Reply:
Moved to root.